src/components/scene/renderer.js

```diff
@@ -13,6 +13,8 @@ var warn = debug('components:renderer:warn');
 module.exports.Component = register('renderer', {
   schema: {
     antialias: {default: 'auto', oneOf: ['true', 'false', 'auto']},
+    maxCanvasWidth: {default: -1},
```

Comment:
1920 x 1080 might be a reasonable default?

Reply:
Shouldn't the default be uncapped?

Reply:
I think it might be good to limit as default to 1920x1080 because higher might be overkill and degrade performance across the board. I think all A-Frame apps should be capped because so many people have Retina screens. For example, ShaderToy caps to a very small resolution.

VR mode is the more important part, I think it's okay to cap what is essentially the preview. What do you think?

Reply:
Alright

Reply:
So I added the defaults here, since it avoids the warning for an undefined attribute – but the default value doesn't have an effect, hence this line:

`width: rendererAttr.maxCanvasWidth ? parseInt(rendererAttr.maxCanvasWidth) : -1`

Is that a problem?